workflow/common/common.go

```diff
@@ -30,6 +30,9 @@ const (
 	// was scheduled to run by CronWorkflow.
 	AnnotationKeyCronWfScheduledTime = workflow.WorkflowFullName + "/scheduled-time"
 
+	AnnotationKeyWorkflowName = workflow.WorkflowFullName + "/workflow-name"
```

Comment:
All the other entries in this constant have a comment explaining what they are for. It would be good to add comments for these new entries.